buildSrc/src/main/java/Dependencies.kt

```diff
@@ -11,7 +11,7 @@ object Versions {
     const val kotlinxSerialization = "1.2.2"
     const val kotlinxHtmlJs = "0.7.3"
 
-    const val compose = "1.0.5"
+    const val compose = "1.1.0-beta01"
```

Comment:
Just curious what changes required this version?

Reply:
Your baseline commit!

Reply:
Ah, ok.....can you rebase/update to use current dependencies....I was going to try and keep this repo on stable Jetpack Compose (for as long as possible anyway!)